Makefile

```diff
@@ -186,7 +190,7 @@ rover: $(ROVER)
 $(S2I_CONTEXT)/Roverfile.lock : $(S2I_CONTEXT)/Roverfile
 	$(ROVER) lock --roverfile=$(S2I_CONTEXT)/Roverfile
 
-dependencies: $(ROVER) $(S2I_CONTEXT)/Roverfile.lock
+dependencies: $(ROVER) $(S2I_CONTEXT)/Roverfile.lock cpan carton
```

Comment:
Lua dependencies don't depend on cpan or carton.

But there could be new task that depends on this one + carton to install everything.

Reply:
agree